pkg/catalog/manager/traverse.go

```diff
@@ -243,9 +255,20 @@ func (m *Manager) traverseAndUpdate(helm *helmlib.Helm, commit string, cmt *Cata
 	} else if clusterCatalog != nil {
 		clusterCatalog.Catalog = *catalog
 	}
+	/*catalog completed upgrade calls and diskcache so this is set true for all catalogs
+	and ensures backoff at norman level
+	*/
+	v3.CatalogConditionUpgraded.True(catalog)
+	v3.CatalogConditionDiskCached.True(catalog)
 	cmt.catalog = catalog
 	cmt.projectCatalog = projectCatalog
 	cmt.clusterCatalog = clusterCatalog
+	if len(createErrors) > 0 {
+		if _, err := m.updateCatalogInfo(cmt, catalogType, "", false, true); err != nil {
+			return err
+		}
+		return errors.Errorf("failed to create templates. Multiple error occurred: %v", createErrors)
```

Comment:
change errors.wrap

Reply:
this could potentially be a lot of info, might want to separate by new lines and maybe only show the actual errors in debug mode?

Reply:
The error information needs to be returned to the user instead of only during debug. Errors are separated by create, update and delete errors instead of new lines. Ensuring the backoff is happening we are reducing the space that logs will take up.